test/asana/helpers/test_extract_task_fields_from_pull_request.py

```diff
@@ -454,14 +454,14 @@ def test_completed_is_false_if_pr_is_closed_and_was_approved_before_merging_but_
         self.assertEqual(False, task_fields["completed"])
 
     @patch(
-        "src.github.logic.SGTM_FEATURE__FOLLOWUP_REVIEW_GITHUB_USERS", [followup_bot]
+        "src.github.logic.SGTM_FEATURE__FOLLOWUP_REVIEW_GITHUB_USERS", [followup_bot.id()]
     )
     def test_completed_is_false_if_pr_is_closed_and_was_approved_by_followup_user(self):
         pull_request = build(
             builder.pull_request()
             .closed(True)
             .merged(True)
-            .merged("2020-01-13T14:59:59Z")
```

Comment:
I'm surprised that mypy didn't fail here... Since `merged` expects a `bool`, I would expect that to have been an error - I need to check up on that, since I do run mypy during CircleCI builds, which would have helped catch this

Reply:
`mypy` is quite happy here - I ran it on the `src/` and `test/` directories.

Reply:
Yeah, same - but it doesn't make sense to me - I know I'm on an old version of mypy but even on the newest version that error doesn't show up (but a bunch of other warnings do)